raft/src/test/java/org/apache/kafka/raft/RaftClientTestContext.java

```diff
@@ -709,6 +709,9 @@ static void verifyLeaderChangeMessage(
         assertEquals(leaderId, leaderChangeMessage.leaderId());
         assertEquals(voters.stream().map(voterId -> new Voter().setVoterId(voterId)).collect(Collectors.toList()),
             leaderChangeMessage.voters());
+        assertEquals(voters.stream().map(voterId -> new Voter().setVoterId(voterId)).collect(Collectors.toSet()),
```

Comment:
Hmm.. It's curious that we can always rely on the full voter set. Do we not have any test cases where we have not received votes from all members?

Reply:
Yeah at least I couldn't find one. This method `verifyLeaderChangeMessage` is invoked only from 2 methods, one being `testInitializeAsCandidateAndBecomeLeader` and the other one is `testLeaderAppendSingleMemberQuorum`. In both these, the full vote is granted. Plz let me know if you want another test which takes care of the case where not all votes are received.

Reply:
Yeah, if you don't mind, it seems like a gap. Thanks!

Reply:
@hachikuji , I have added a test case where i initialize a quorum of 3 and got votes from majority. Needed to tweak a couple of methods as some of them were having hardcoded values of 1 on the expected value. Plz review.